circleguard/visualizer.py

```diff
@@ -671,7 +668,7 @@ def play_reverse(self):
         self._update_speed()
 
     def _update_speed(self):
-        self.renderer.clock.change_speed(float(self.speed_label.text()) * self.renderer.play_direction)
+        self.renderer.clock.change_speed(float(self.speed_label.text())*self.renderer.play_direction)
```

Comment:
![](https://i.imgur.com/6BxX5Mf.png)
and same through the whole file

Reply:
might have missed some but there's my pass